daemon/graphdriver/overlay2/overlay.go

```diff
@@ -213,7 +216,12 @@ func parseOptions(options []string) (*overlayOptions, error) {
 			if err != nil {
 				return nil, err
 			}
-
+		case "overlay2.default_size":
```

Comment:
Added the documentation portion.

Made the change to the option being called as overlay2.size instead of overlay2.default_size